src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_IIfStatement.cs

```diff
@@ -869,6 +869,295 @@ private static void A(bool flag, int number)
+                          ILiteralOperation (OperationKind.Literal, Type: System.Int32, Constant: 5) (Syntax: '5')
+                          InConversion: CommonConversion (Exists: True, IsIdentity: True, IsNumeric: False, IsReference: False, IsUserDefined: False) (MethodSymbol: null)
+                          OutConversion: CommonConversion (Exists: True, IsIdentity: True, IsNumeric: False, IsReference: False, IsUserDefined: False) (MethodSymbol: null)
+                    Initializer: 
+                      null
+                  InConversion: CommonConversion (Exists: True, IsIdentity: True, IsNumeric: False, IsReference: False, IsUserDefined: False) (MethodSymbol: null)
+                  OutConversion: CommonConversion (Exists: True, IsIdentity: True, IsNumeric: False, IsReference: False, IsUserDefined: False) (MethodSymbol: null)
+";
+            var expectedDiagnostics = new DiagnosticDescription[] {
+                // file.cs(7,6): error CS1003: Syntax error, 'if' expected
+                //     {
+                Diagnostic(ErrorCode.ERR_SyntaxError, "").WithArguments("if", "else").WithLocation(7, 6),
+                // file.cs(7,6): error CS1003: Syntax error, '(' expected
+                //     {
+                Diagnostic(ErrorCode.ERR_SyntaxError, "").WithArguments("(", "else").WithLocation(7, 6),
+                // file.cs(7,6): error CS1525: Invalid expression term 'else'
+                //     {
+                Diagnostic(ErrorCode.ERR_InvalidExprTerm, "").WithArguments("else").WithLocation(7, 6),
+                // file.cs(7,6): error CS1026: ) expected
+                //     {
+                Diagnostic(ErrorCode.ERR_CloseParenExpected, "").WithLocation(7, 6),
+                // file.cs(7,6): error CS1525: Invalid expression term 'else'
+                //     {
+                Diagnostic(ErrorCode.ERR_InvalidExprTerm, "").WithArguments("else").WithLocation(7, 6),
+                // file.cs(7,6): error CS1002: ; expected
+                //     {
+                Diagnostic(ErrorCode.ERR_SemicolonExpected, "").WithLocation(7, 6)
+            };
+
+            VerifyOperationTreeAndDiagnosticsForTest<IfStatementSyntax>(source, expectedOperationTree, expectedDiagnostics);
+        }
+
+        [CompilerTrait(CompilerFeature.IOperation)]
+        [Fact, WorkItem(27866, "https://github.com/dotnet/roslyn/issues/27866")]
+        public void IIfstatementWithDoubleElse()
+        {
+            string source = @"
+using System;
+
+class P
+{
+    private void Op()
+    {
+    }
+
+    private void M(bool flag)
+    {
+        /*<bind>*/if (flag)
+        {
+            Op();
+        }
+        else
+        {
+            Console.WriteLine(flag);
+        }
+        else 
+        {
+            Console.WriteLine(!flag);
+        }
+/*</bind>*/    }
+}
+";
+            string expectedOperationTree = @"
```

Comment:
>expectedOperationTree [](start = 19, length = 21)

Why aren't the second `else` and its block appearing in the output?

Reply:
I tried to bind 2 operation statements. In 3016d83 I wrapped them in a block statement.